packages/react-core/src/demos/DataList/examples/DataListPagination.tsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import {
+  Button,
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  Flex,
+  FlexItem,
+  MenuToggle,
+  MenuToggleCheckbox,
+  OverflowMenu,
+  OverflowMenuControl,
+  OverflowMenuItem,
+  PageSection,
+  PageSectionVariants,
+  Pagination,
+  Text,
+  TextContent,
+  TextVariants,
+  Toolbar,
+  ToolbarItem,
+  ToolbarContent,
+  PaginationVariant
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+
+import CodeBranchIcon from '@patternfly/react-icons/dist/esm/icons/code-branch-icon';
+import CodeIcon from '@patternfly/react-icons/dist/esm/icons/code-icon';
+import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
+import { rows, getRandomInteger } from '@patternfly/react-table/dist/esm/demos/sampleData';
```

Comment:
Wondering whether we should be importing these from the Table package. Rather than the `rows`, we could just repeat the same 3 items ("patternfly", "patternfly-elements", and "patternfly-unified-design-kit" with whatever applicable icons) however many times to get the pagination working. Would probably require having something to build the array and then assigning it to a `rows` variable internal to the demo.

If we did want to just use the `rows` data, we could at least avoid needing to import getRandomInteger and just destructure the `lastModified` property on line 132

@gitdallas wdyt?

Reply:
😆 beat you by a couple minutes with the same concern - i guess we validate each others concerns then